README.adoc

```diff
@@ -42,10 +42,10 @@ You can take a look at the link:sonar-project.properties[sonar-project.propertie
 = Documentation
 
 - https://docs.sonarqube.org/latest/analysis/languages/cfamily/[Documentation of the C, C++ and Objective-C plugin and its Build Wrapper]
-- https://sonarcloud.io/documentation/getting-started/azure-devops/[Documentation of using SonarCloud with Azure DevOps]
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/#analysis-cache[Configuring C, C++ and Objective-C analysis cache]
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/#multithreaded-code-scan[Configuring multithreaded execution]
-- https://sonarcloud.io/documentation/analysis/languages/cfamily/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
+- https://docs.sonarcloud.io/getting-started/azure-devops/[Documentation of using SonarCloud with Azure DevOps]
+- https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[Configuring C, C++ and Objective-C analysis cache]
+- https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#multithreaded-code-scan[Configuring multithreaded execution]
+- https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
```

Comment:
And still not related to what you just did, could you also correct the typo l.57 (reffer -> refer) ? Thanks!

Reply:
done.